aws/resource_aws_elasticache_replication_group.go

```diff
@@ -941,20 +883,10 @@ func deleteElasticacheReplicationGroup(replicationGroupID string, conn *elastica
 	}
 
 	if err != nil {
-		return fmt.Errorf("error deleting Elasticache Replication Group: %w", err)
+		return fmt.Errorf("error deleting ElastiCache Replication Group: %w", err)
 	}
 
-	log.Printf("[DEBUG] Waiting for deletion: %s", replicationGroupID)
-	stateConf := &resource.StateChangeConf{
-		Pending:    []string{"creating", "available", "deleting"},
-		Target:     []string{},
-		Refresh:    cacheReplicationGroupStateRefreshFunc(conn, replicationGroupID, []string{}),
-		Timeout:    40 * time.Minute,
-		MinTimeout: 10 * time.Second,
-		Delay:      30 * time.Second,
-	}
-
-	_, err = stateConf.WaitForState()
+	_, err = waiter.ReplicationGroupDeleted(conn, replicationGroupID, timeout)
 	return err
```

Comment:
Recommend wrapping error:

```suggestion
	if err != nil {
		return fmt.Errorf("error waiting for ElastiCache Replication Group (%s) to delete: %w", err)
	}

	return nil
```